### PR TITLE
Add support for multiple clusters

### LINKED ISSUE #245: modify database management script to include private key of the user
In the dabase we should keep entries necessary to login to the cluster:
 - cluster name [i.e. ares.cyfronet.pl]
 - username [i.e. plgXXX]
 - private key

### PR DESCRIPTION
**Pull request by p1003**

### CODE REVIEW

**Comment by grzanka** [comment]
Remember to update API documentation once this PR is merged. As for now it looks OK

File: yaptide/admin/db_manage.py [hunk 1]
```diff
@@ -68,7 +69,7 @@ def list_users(**kwargs):
     ResultSet = ResultProxy.fetchall()
     click.echo(f"{len(ResultSet)} users in DB:")
     for row in ResultSet:
-        click.echo(f"Login {row.username} ; Passw ...{row.password_hash[-10:]} ; Proxy {proxy_hash(row.grid_proxy)}")
+        click.echo(f"Login {row.username}; Passw ...{row.password_hash[-10:]}; Proxy {proxy_hash(row.grid_proxy)}")
```
Comment (grzanka) [inline yaptide/admin/db_manage.py]: Get rid of proxy hash or replace it with key hash

Comment (p1003) [reply yaptide/admin/db_manage.py]: ok

**Comment by grzanka** [comment]
This seems not to work with recent deploy on C3

**Comment by p1003** [comment]
I will check this

**Review by grzanka — APPROVED** [review]
ok

**Pull request merged**